.github/workflows/backend-cd.yml

```
@@ -0,0 +1,40 @@
+      - name: 도커허브에 이미지 푸시
+        run: docker push ${{ secrets.DOCKER_USER_NAME }}/${{ secrets.DOCKER_IMAGE_NAME }}-be:latest
+      - name: AWS EC2에 ssh 접속 후 배포
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.AWS_IP }}
+          port: 22
+          username: ubuntu
+          key: ${{ secrets.AWS_KEY }}
+          script: |
+            docker pull ${{ secrets.DOCKER_USER_NAME }}/${{ secrets.DOCKER_IMAGE_NAME }}-be:latest
+            docker-compose --env-file .env.production up -d
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)